app/shared/directives/noMobileSupport/noMobileSupport.html

```diff
@@ -0,0 +1,7 @@
+<div class="no-mobile-support">
+	<div class="jumbotron">
+		<p class="lead">IPA does not currently support mobile devices.</p>
+		<br />
+		<p class="lead">Please reload this page on a computer.</p>
```

Comment:
'reload' 'on a computer' is a bit of an abuse of language. Let's go with 'Please visit' instead.